### PR TITLE
Cleaning up metadata and redirects

### PR DESCRIPTION
**Pull request by CamSoper**

### CODE REVIEW

**Comment by CamSoper** [comment]
@mairaw I'd add you as a reviewer, but I don't think you're in the Azure org.  Can you please give this a sanity check before I merge?

Thanks!

File: docfx.json [hunk 1]
```diff
@@ -149,9 +149,9 @@
         "api/Microsoft.WindowsAzure.Storage*.yml": "storage",
         "api/Microsoft.Azure.Management.StreamAnalytics*.yml": "stream-analytics",
         "api/Microsoft.Azure.Management.TrafficManager*.yml": "traffic-manager",
-        "api/Microsoft.Azure.Management.Compute*.yml": "big-compute",
-        "api/Microsoft.Practices.EnterpriseLibrary.TransientFaultHandling*.yml": "Other",
-        "api/Microsoft.Azure.Management.Graph.RBAC.Fluent*.yml": "cosmos db"        
+        "api/Microsoft.Azure.Management.Compute*.yml": "multiple",
```
[the remark this answers is not in the document]
Comment (CamSoper) [reply docfx.json]: @mairaw sure, done!  🙂

File: docfx.json [hunk 2]
```diff
@@ -95,63 +95,63 @@
              "xml/**/*": "None"
       },
       "ms.service": {
-        "api/Microsoft.Identitymodel*.yml": "active-directory",
-        "api/Microsoft.Identity*.yml": "active-directory",
+        "api/DeviceExplorer*.yml": "iot-hub",
+        "api/KeyVault.TestFramework*.yml": "key-vault",
+        "api/Microsoft.ApplicationInsights*.yml": "application-insights",
+        "api/Microsoft.Azure.Batch*.yml": "batch",
```
Comment (mairaw) [inline docfx.json]: ah! so much better. so now I can compare with that folder and see missing paths like Microsoft.Azure.ApplicationInsights 😄 
Do you wanna double check that list?

Comment (CamSoper) [reply docfx.json]: @mariraw Done. I think the whole thing looks good, you?

**Pull request merged**